packages/tradershub/src/features/cfd/flows/MT5/AddedMT5AccountsList/AddedMT5AccountsList.tsx

```diff
@@ -40,11 +41,12 @@ const AddedMT5AccountsList = ({ account }: { account: THooks.MT5AccountsList })
                         colorStyle='black'
                         disabled={jurisdictionStatus.is_failed || jurisdictionStatus.is_pending}
                         onClick={() => {
-                            history.push('/cashier/transfer');
+                            if (isVirtual) show(<TopUpModal account={account} platform={CFDPlatforms.MT5} />);
+                            else history.push('/cashier/transfer');
```

Comment:
should'nt show transfer modal here?

Reply:
@mahdiyeh-deriv yeah, you are right, but we already have this push to cashier/transfer so I've just leave it there, because we don't have Transfer funds modal in tradershub package at this moment. We'll change it when this modal will be created

Reply:
@sergei-deriv let's remove the redirection and put a comment like what you've did in this file
packages/tradershub/src/features/cfd/flows/CTrader/AddedCTraderAccountsList/AddedCTraderAccountsList.tsx